plugin/evm/orderbook/memory_database.go

```diff
@@ -480,18 +597,26 @@ func (db *InMemoryDatabase) GetLongOrders(market Market, lowerbound *big.Int, bl
 
 func (db *InMemoryDatabase) getLongOrdersWithoutLock(market Market, lowerbound *big.Int, blockNumber *big.Int, shouldClean bool) []Order {
 	var longOrders []Order
-	for _, order := range db.OrderMap {
-		if order.PositionType == LONG && order.Market == market && (lowerbound == nil || order.Price.Cmp(lowerbound) >= 0) {
-			if shouldClean {
-				if _order := db.getCleanOrder(order, blockNumber); _order != nil {
-					longOrders = append(longOrders, *_order)
-				}
-			} else {
-				longOrders = append(longOrders, *order)
+
+	marketOrders := db.LongOrders[market]
+	if marketOrders == nil {
+		return longOrders
+	}
```

Comment:
nil check is redundant, please verify and if true remove from everywhere in this file. easier on the eyes

Reply:
removed